flutter/example/integration_test/launch_test.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/widgets.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:sentry_flutter/sentry_flutter.dart';
+import 'package:sentry_flutter_example/main.dart';
+
+void main() {
+  final binding = IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+  binding.framePolicy = LiveTestWidgetsFlutterBindingFramePolicy.fullyLive;
+
+  testWidgets('setup sentry and render app', (tester) async {
+    await setupSentry(() async {
+      await tester.pumpWidget(DefaultAssetBundle(
+        bundle: SentryAssetBundle(enableStructuredDataTracing: true),
+        child: MyApp(),
+      ));
+      await tester.pumpAndSettle();
+    });
+
+    // Find any UI element and verify it is present.
+    expect(find.text('Open another Scaffold'), findsOneWidget);
```

Comment:
That would only catch issues with the start itself, we need to add more cases for calling all the native bridge channels, for example, all the scope sync methods, capture messages, etc.

Reply:
I have added tests by calling native bridge directly. Opted to not call though the public sentry API, as we are handling/logging errors from native channel side and i think are also not awaiting until it is finished in the sync APIs. WDYT of this approach?

Reply:
> i think are also not awaiting until it is finished in the sync APIs. WDYT of this approach?

Not sure if I understand this statement, can you elaborate?

Reply:
Only calling public API of the Flutter SDK now. Now we don't call every native method through the bridge anymore, as many of them are "buried" in integrations.